### PR TITLE
Feedback from team: Textual updates, bug fixes and refactor

### PR DESCRIPTION
**Pull request by robinpiets**
Features:
- Changed text in Conclusion Alert
- Changed text for Loading component
- Hide PermitButton in case of contactConclusion
- Refactored what to print and what not (Re-added the data and URL on @media print)

### CODE REVIEW

**Review by svenjens — APPROVED** [review]

File: apps/client/src/atoms/HideForPrint.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+export default styled.div`
```
Comment (afjlambert) [inline apps/client/src/atoms/HideForPrint.js:3]: 👍

File: apps/client/src/components/Layouts/BaseLayout.tsx [hunk 2]
```diff
@@ -38,12 +39,16 @@ function BaseLayout({ children, checker, heading }: BaseLayoutProps) {
...
+                <FormTitle style={{ maxWidth: "inherit" }}>{title}</FormTitle>
+              )}
               {children}
             </Content>
           </Column>
         </Row>
+      </ContentContainer>
 
+      <HideForPrint>
         <HiddenDebugInfo title="Environment">
```
Comment (afjlambert) [inline apps/client/src/components/Layouts/BaseLayout.tsx:50]: Maybe good to do similar approach for HiddenDebugInfo as HideForPrint sometime in the future. Put it in an atom and rename.

File: apps/client/src/components/Layouts/BaseLayout.tsx [hunk 3]
```diff
@@ -38,12 +39,16 @@ function BaseLayout({ children, checker, heading }: BaseLayoutProps) {
             }}
           >
             <Content>
-              {title && <FormTitle>{title}</FormTitle>}
+              {title && (
+                <FormTitle style={{ maxWidth: "inherit" }}>{title}</FormTitle>
```
Comment (afjlambert) [inline apps/client/src/components/Layouts/BaseLayout.tsx]: Inline style? Maybe create an atom instead?
Even better would be to fix FormTitle itself.

File: apps/client/src/components/PrintDetails.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+export default () => (
+  <PrintOnly>
+    <Heading forwardedAs="h3">Pagina</Heading>
+    <Paragraph fontSize={12}>{window.location.href}</Paragraph>
+
+    <Heading forwardedAs="h3">Datum</Heading>
+    <Paragraph>{currentDateTime} uur.</Paragraph>
+  </PrintOnly>
+);
```
Comment (afjlambert) [inline apps/client/src/components/PrintDetails.js:25]: Nice one!

**Review by afjlambert — CHANGES_REQUESTED** [review]

File: apps/client/src/components/Layouts/BaseLayout.tsx (hunk 2 above)
Comment (robinpiets) [reply apps/client/src/components/Layouts/BaseLayout.tsx:50]: I have it noted on the refactor list

File: apps/client/src/components/Layouts/BaseLayout.tsx (hunk 3 above)
Comment (robinpiets) [reply apps/client/src/components/Layouts/BaseLayout.tsx]: Good point. I've changed this in ASC-UI in this [PR](https://github.com/Amsterdam/amsterdam-styled-components/pull/1086).

**Review by afjlambert — APPROVED** [review]

**Pull request merged**